### PR TITLE
dotCMS/core#23739 Close the gap in the dojo dialogs title

### PR DESCRIPTION
**Pull request by zulqarnainvd**
### Proposed Changes
* Close the gap in the dojo dialogs title and content

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **
Screenshot from 2023-01-30 18-11-28|Screenshot from 2023-01-30 18-10-10

### CODE REVIEW

**Comment by zulqarnainvd** [comment]
Dojo Dialog adding a width automatically of it's content and title that's why I use `!important`

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/libs/dojo-theme/scss/backend/dot-admin/portlets/_content-search.scss [hunk 1]
```diff
@@ -27,6 +27,11 @@
 .content-search__show-query-dialog {
     .dijitDialogTitleBar {
         z-index: 1;
+        width: inherit !important;
+    }
+
+    .dijitDialogPaneContent {
+        width: inherit !important;
```
[the remark this answers is not in the document]
Comment (zulqarnainvd) [reply core-web/libs/dojo-theme/scss/backend/dot-admin/portlets/_content-search.scss:34]: Noted

[the remark this answers is not in the document]
Comment (zulqarnainvd) [reply core-web/libs/dojo-theme/scss/backend/dot-admin/portlets/_content-search.scss]: updated

File: core-web/libs/dojo-theme/scss/backend/dot-admin/portlets/_content-search.scss [hunk 2]
```diff
@@ -24,10 +24,8 @@
     height: 100%;
 }
 
-.content-search__show-query-dialog {
-    .dijitDialogTitleBar {
-        z-index: 1;
-    }
+#queryResults {
+    padding: 16px;
 }
```
Comment (fmontes) [inline core-web/libs/dojo-theme/scss/backend/dot-admin/portlets/_content-search.scss:29]: Why do we need this?

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/libs/dojo-theme/scss/backend/dot-admin/portlets/_content-search.scss (hunk 2 above)
Comment (zulqarnainvd) [reply core-web/libs/dojo-theme/scss/backend/dot-admin/portlets/_content-search.scss:29]: I need space under the query box on content search dialog

**Pull request merged**